src/main/java/com/userservice/Controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package com.userservice.Controller;
+
+import com.userservice.Entity.Users;
+import com.userservice.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+@RestController
+@RequestMapping("/api")
+public class UserController {
+    @Autowired
+    private UserService userService;
+    @PostMapping("/create")
+    public ResponseEntity<?>createUser(@RequestBody Users users){
+        return userService.create(users);
+    }
+    @GetMapping("getUser/{username}")
```

Comment:
use the phone number with the country code to get the user details. Also, create one more function to get the user details using the id. In microservice architecture, multiple services will be calling each other and probably they will be using ID instead of the phone number. So getting details with phone numbers and with ID are required.